### DISCUSSION TITLE
My dropdown doesn't work

### DISCUSSION THREAD

**Discussion by monirhabderabby** [themesberg/flowbite#172] [Q&A]
Hi Dear!
I am facing an issue in the dropdown. I install the and setup my tailwind properly and also react flowbite. I am trying to use dropdown components in my react project. but somehow my dropdown didn't work. 

I did also try to use "navbar with dropdown" but I got the same problem that is dropdown isn't work in my project. 

My install and step process is below:
> npm install -D tailwindcss postcss autoprefixer
> npx tailwindcss init -p
> npm i flowbite flowbite-react

my config file are: 
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./src/**/*.{js,jsx,ts,tsx}",
    'node_modules/flowbite-react/**/*.{js,jsx,ts,tsx}'
  ],
  theme: {
    extend: {},
  },
  plugins: [
    require('flowbite/plugin')
  ],
}

I added this to  index.css file are:
@tailwind base;
@tailwind components;
@tailwind utilities;

I added this to index.js file:
import 'flowbite';

I added this to index.html file before the ending body tag:
<script src="../path/to/flowbite/dist/flowbite.js"></script> 

Is there any missing from my setup?
How can I solve my dropdown issue?

Thanks
Monir Hossain

**Comment by adriDiazz** [comment]
have you solve the problem, im getting the same error

**Reply by monirhabderabby** [reply]
Yes, I solved my problem.

I just solved my problem using components from https://flowbite-react.com/

let's connect: https://www.linkedin.com/in/monirhabderabby/

Thanks
Monir Hossain

**Reply by semiautomatix** [reply]
Seems that a solution needs to be created for Solidjs. I don't have time for that, so moving on.

**Reply by Varun-IKM** [reply]
I'm also facing the same. Dropdown menu is not working.
